frontend/rolecall/src/app/cast/cast-drag-and-drop.component.ts

```diff
@@ -343,13 +355,49 @@ export class CastDragAndDrop implements OnInit {
 
   decrementDancerCount(positionIndex: number) {
     this.castPositions[positionIndex].castRows.pop();
-    this.castPositions[positionIndex].castSize -= 1;
+    this.castPositions[positionIndex].dancerCount -= 1;
   }
 
   incrementDancerCount(positionIndex: number) {
     this.castPositions[positionIndex].castRows.push({
-      subCastDancers: new Array(CAST_COUNT),
+      subCastDancers: new Array(this.castCount),
     });
-    this.castPositions[positionIndex].castSize += 1;
+    this.castPositions[positionIndex].dancerCount += 1;
+  }
+
+  private changeCastCount(change: number) {
```

Comment:
Currently it sounds like it will SET the count to the given value rather than increment/decrement, I recommend this for better readability:
```
/** Increments cast count by given amount, pass in negative value to decrement. */
private incrementCastCount(amount: number) {
```

Reply:
Done.